src/Botan.php

```diff
@@ -187,17 +187,17 @@ public static function track(Update $update, $command = '')
                 ]
             );
 
-            $response = (string) $response->getBody();
+            $result = (string) $response->getBody();
         } catch (RequestException $e) {
-            $response = ($e->getResponse()) ? (string) $e->getResponse()->getBody() : '';
+            $result = $e->getMessage();
         } finally {
-            $responseData = json_decode($response, true);
+            $responseData = json_decode($result, true);
 
             if ($responseData['status'] !== 'accepted') {
-                if (!empty($response)) {
-                    TelegramLog::debug("Botan.io stats report failed, API reply: $response");
+                if (!empty($result)) {
```

Comment:
Well, if you're doing it like this, might as well shorten it all the way 😃
```php
TelegramLog::debug('Botan.io stats report failed: ' . ($result ?: 'empty response') . "\n\n");
```

*(same for the bottom one)*